src/styles/index.scss

```diff
@@ -10,5 +10,6 @@ body {
 @import 'text-field';
 @import 'spinner';
 @import 'notification';
+@import 'cookies-notification';
```

Comment:
Ok.. this is a specific rule, but for me, this is a good idea. We should write all imports in alphabetical order.